src/App.js

```diff
@@ -395,6 +415,8 @@ class Board extends React.Component {
     let amountRemove = 46;
     if(mode === "easy") {
       amountRemove = 15;
+    } else if(mode === "medium") {
```

Comment:
I think difficulty values should be setup as globals, maybe as enums?

Reply:
So, instead of mode, it's difficulty values be global?

Reply:
done